code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1055,6 +1055,10 @@
 	color = "#A5F0EE" // rgb: 165, 240, 238
 	taste_description = "sourness"
 	reagent_weight = 0.6 //so it sprays further
+	
+/datum/reagent/space_cleaner/on_mob_life(mob/living/carbon/M)
+	M.adjustToxLoss(0.25)
```

Comment:
ingest-only please, or else space cleaner foam and space cleaner sprays will deal minor amounts of tox damage to people

Reply:
given the reference it should also be on inject

Reply:
ah, fair point

[ingest or inject]-only, then, please